README.md

```diff
@@ -2,10 +2,10 @@
 
 This is the source for https://docs.snowplow.io/docs.
 
-* [Contributing](#contributing)
-* [How to preview locally](#how-to-preview-locally)
-* [Organizing content](#organizing-content)
-* [Formatting content](#formatting-content)
+- [Contributing](#contributing)
```

Comment:
We should probably exclude all Markdown as opposed to just `/docs`? For consistency.